jdk-profiles/pom.xml

```diff
@@ -43,5 +43,33 @@
                 <jdk>[9,)</jdk>
             </activation>
         </profile>
+        <!-- 
+            TODO does not really belong here but this is a pom without activatedByDefault profiles 
+            that would be deactivated by mistake
+        -->
+        <profile>
+            <id>buildFaster</id>
+            <properties>
+                <maven.scalastyle.skip>true</maven.scalastyle.skip>
+                <maven.scaladoc.skip>true</maven.scaladoc.skip>
+                <rat.skip>true</rat.skip>
```

Comment:
Do all the github workflow builds now build this way, and are we relying on the blossom build to catch scalastyle, scaladoc, and licensing issues?

Reply:
it turns out this is not working at all as intended right now, so I am going to table this change, and resort to using just -D properties